### PR TITLE
[APP-7026] Handle rfkill, no-wifi-device cases

### PR DESCRIPTION
**Pull request by Otterverse**
This fixes two cases... the first, in the bug, is when the wifi is blocked by rfkill (aka, "airplane mode".) This has recently (past week) become the default for new Raspberry Pi image installs unless the user specifically adds a wifi network during the image creation process.

The second case is for devices that have no wifi adapters at all. Previously, results were undefined, and provisioning would continue trying to execute, scan, start hotspots, etc. even if that wasn't possible. Now the lack of wifi hardware causes provisioning to self-disable until the next restart of agent.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.23.1
 
 require (
 	github.com/Masterminds/semver/v3 v3.3.0
-	github.com/Otterverse/gonetworkmanager/v2 v2.2.0
+	github.com/Otterverse/gonetworkmanager/v2 v2.2.1
```
Comment (Otterverse) [inline go.mod:7]: Small patch over there to add GetPropertyRadioFlags()

File: subsystems/provisioning/setup.go [hunk 2]
```diff
@@ -139,3 +141,24 @@ func (w *Provisioning) initDevices() error {
...
 }
+
+func (w *Provisioning) enableWifi(ctx context.Context) error {
+	if err := w.nm.SetPropertyWirelessEnabled(true); err != nil {
+		return err
+	}
+
+	timeoutCtx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+	for {
```
Comment (Otterverse) [inline subsystems/provisioning/setup.go:152]: Without this loop the first scan fails and throws an error, as it takes a couple seconds for the radio to power on.

File: utils.go [hunk 3]
```diff
@@ -332,12 +332,3 @@ func ConvertAttributes[T any](attributes *structpb.Struct) (*T, error) {
 
 	return newConfig, nil
 }
-
-func Sleep(ctx context.Context, timeout time.Duration) bool {
```
Comment (Otterverse) [inline utils.go:336]: This is just drive-by cleanup. Nothing uses this anymore.

File: subsystems/provisioning/provisioning.go [hunk 4]
```diff
@@ -124,6 +124,18 @@ func (w *Provisioning) getNM() (gnm.NetworkManager, error) {
 		return nil, nmErr
 	}
 
+	flags, err := nm.GetPropertyRadioFlags()
+	if err != nil {
```
Comment (ale7714) [inline subsystems/provisioning/provisioning.go:128]: 👍

**Review by ale7714 — APPROVED** [review]
